src/main/java/frc/robot/poseestimator/Pose2dArrayEntryValue.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.poseestimator;
+
+public enum Pose2dArrayEntryValue {
+
+	X_VALUE(0),
+	Y_VALUE(1),
+	ROTATION_VALUE(2);
+
+	private final int entryValue;
+
+	Pose2dArrayEntryValue(int entryValue) {
+		this.entryValue = entryValue;
+	}
+
+	public int getEntryValue() {
+		return entryValue;
+	}
+
+	public static final int POSE_ARRAY_LENGTH = 3;
+
+}
```

Comment:
and when you want to access a value in that array, like the Y value, instead of writing:
pose[1], you can write: 
pose[Pose2dArrayEntryValue.Y_VALUE.getEntryValue()]